### DISCUSSION TITLE
Why modules/fetures execute twice having only the `Default` tenant?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#14417] [Q&A]
I set only the `Default` tenant via `appsettings.json` but feature startup executes twice

``` JSON
{
  "OrchardCore": {
    "Default": {
      "State": "Running",
      "RequestUrlHost": null,
      "RequestUrlPrefix": null,
      "Features": [
        "OrchardCore.Mvc.HelloWorld",
        "OrchardCore.Mvc.HelloFSharp.Feature"
      ]
    }
  },
}
```

**Comment by jtkech** [comment]
> but feature startup executes twice

That's normal, for the default tenant we build an isolated context for an hosted service which is used to sync running instances if Redis is enabled and also the Tenants.Distributed feature.
